hal-x86_64/src/serial.rs

```diff
@@ -248,6 +278,39 @@ impl<'a> Lock<'a> {
 }
 
 impl<B> Lock<'_, B> {
+    pub fn check_interrupt_type(&mut self) -> io::Result<Option<Pc16550dInterrupt>> {
+        // IIR bits 0 through 3 describe what happened to produce an interrupt, with bits 4 and 5
+        // always 0, and bits 6 and 7 set to 1 if `FCR0=1`.
```

Comment:
what does FCR0 mean? i assume it's for Flow ContR0l...?

Reply:
add a bit more in the module docs because i forgot where this stuff even is defined or how the datasheet talks about bits (and it doesn't... really clarify itself either), but "FIFO Control Register bit 0" which is a very 1995 way to say it